definitions/features/foreman_server.rb

```diff
@@ -9,9 +9,11 @@ class ForemanServer < ForemanMaintain::Feature
       end
 
       def services
-        [
-          system_service('httpd', 30)
-        ]
+        if feature(:foreman_service)
+          [feature(:foreman_service).service]
+        else
+          [system_service('httpd', 30)]
```

Comment:
Similarly, do I need to handle [line number 30](https://github.com/theforeman/foreman_maintain/pull/321/files#diff-0e91ba9aec5b0d6281275bda1ece9fb5R30)?

Reply:
As per the comments from Eric we are going to have separate foreman_service. In this case is it better to limit the foreman_server using confine block ? This way we wont need to add any conditions of feature(:foreman_service) in foreman_server ?

Reply:
Removed `foreman_service` feature and handled condition in same `foreman_server` feature.